torch/csrc/jit/passes/quantization.cpp

```diff
@@ -458,16 +458,20 @@ void InsertObserversHelper::addIntermediateValuesToSkipObserver(
 void InsertObserversHelper::insertObservers(
     script::Module& module,
     const std::string& method_name) {
-  if (!module_qconfig_map_.count(module._ivalue())) {
-    // the module is added by us, e.g.: observer module
-    return;
-  }
   for (auto& invoked_methods: getInvokedMethods(module, method_name)) {
     auto& invoked_module = std::get<0>(invoked_methods);
     const auto& invoked_method_name = std::get<1>(invoked_methods);
     insertObservers(invoked_module, invoked_method_name);
   }
 
+  auto qconfig_opt = module_qconfig_map_.at(module._ivalue());
```

Comment:
Please add a comment explaining why this needs to happen after iterating through invoked methods (the one you have in the commit message is good).